Doc/library/unittest.rst

```diff
@@ -248,10 +253,12 @@ Command-line options
 .. versionadded:: 3.7
    The command-line option ``-k``.
 
+.. versionadded:: 3.9
```

Comment:
The version on this (and the others below) will need to be updated.

Reply:
done